resources/qml/PrinterSelector/MachineSelectorList.qml

```diff
@@ -32,7 +32,7 @@ Column
             id: networkedPrintersModel
             filter:
             {
-                "type": "machine", "um_network_key": "*", "hidden": "False"
+                "type": "machine", "um_network_key": "*", "hidden": "False", "um_connection_type": "[2,3,4]"
```

Comment:
All references of um_network_key should now be removed from the UI? (There's more places in the UI where it's currently used).

Reply:
That's the idea yeah.

Reply:
Also; Put documentation here regarding what the filtering is actually doing. From this code, I have no idea what 2,3,4 means.